frontend/service/agents/PredictTrader.ts

```diff
@@ -182,11 +195,14 @@ export abstract class PredictTraderService extends StakedAgentService {
   static getStakingContractDetails = async (
     stakingProgramId: StakingProgramId,
     chainId: EvmChainId,
-  ): Promise<StakingContractDetails> => {
+  ): Promise<StakingContractDetails | undefined> => {
     const { multicallProvider } = PROVIDERS[chainId];
+    const stakingTokenProxy =
+      STAKING_PROGRAMS[chainId][stakingProgramId]?.contract;
 
-    const { contract: stakingTokenProxy } =
-      STAKING_PROGRAMS[chainId][stakingProgramId];
+    if (!PREDICT_TRADER_STAKING_PROGRAM_IDS.includes(stakingProgramId)) {
+      return Promise.resolve(undefined);
+    }
```

Comment:
can get check this instead

https://github.com/valory-xyz/olas-operate-app/blob/004c19ba5cc1cf2609d6a456faa351abf3101338/frontend/config/stakingPrograms/gnosis.ts#L28-L44